app/commands/user/insiders_status/activate.rb

```diff
@@ -19,6 +19,7 @@ def call
 
     User::UpdateFlair.(user)
     User::Notification::CreateEmailOnly.defer(user, @notification_key)
+    User::Premium::Update.defer(user)
```

Comment:
```suggestion
    User::Premium::Update.(user)
```

Move this above and undefer it as we want this happening in-band?